webapp/src/components/Metrics/PerformanceAnalysis.tsx

```diff
@@ -184,28 +191,45 @@ const PerformanceAnalysis: React.FC<Props> = ({ jobId, pipeline }) => {
       ...NUMBER_COL_DEF,
       field: "utteranceCount",
       headerName: "Utterance Count",
-      minWidth: 160,
+      minWidth: 150,
     },
     ...ALL_OUTCOMES.map<Column<Row>>((outcome) => ({
       ...NUMBER_COL_DEF,
       field: outcome,
       headerName: OUTCOME_PRETTY_NAMES[outcome],
+      minWidth: 105,
       renderHeader: () => OutcomeIcon({ outcome }),
       valueGetter: ({ row }) => row.outcomeCount[outcome] / row.utteranceCount,
-      valueFormatter: percentageFormatter,
+      renderCell: ({ row }: GridCellParams<undefined, Row>) =>
+        visualBarPercentage(
+          row.outcomeCount[outcome] / row.utteranceCount,
+          theme.palette[OUTCOME_COLOR[outcome]].main
+        ),
     })),
     ...customMetricNames.map<Column<Row>>((metricName) => ({
       ...NUMBER_COL_DEF,
       field: metricName,
       headerName: metricName,
+      minWidth: 105,
       valueGetter: ({ row }) => row.customMetrics[metricName],
-      valueFormatter: percentageFormatter,
+      renderCell: ({ row }: GridCellParams<undefined, Row>) =>
+        visualBarPercentage(
+          row.customMetrics[metricName],
+          theme.palette.info.light
+        ),
     })),
     {
       ...NUMBER_COL_DEF,
       field: "ece",
       headerName: "ECE",
-      valueFormatter: twoDigitFormatter,
+      minWidth: 105,
+      renderCell: ({ value }: GridCellParams) => (
+        <VisualBar
+          formattedValue={(value as number).toFixed(2)}
```

Comment:
It's best to define the more specific type `GridCellParams<number>` than forcing the type with `as number`.
```suggestion
      renderCell: ({ value }: GridCellParams<number>) => (
        <VisualBar
          formattedValue={value.toFixed(2)}
```